deploy/helm/kubecf/values.yaml

```diff
@@ -10,19 +11,6 @@ system_domain: ~
 #            logs:
 #              addr: kubecf-log-api:8082
 #
-#  Eirini Persistence Broker setup example:
```

Comment:
I'm removing this chunk because we're not using BOSH releases for this anymore, and we _do_ have defaults at the bottom of the file.

Reply:
👍